### PR TITLE
anchor node version to 5.6.0

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
https://github.com/Financial-Times/ft-labs/issues/2

### CODE REVIEW

**Comment by AdaRoseCannon** [comment]
When the tests pass I will push this to heroku -test

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "license": "",
   "description": "Perf Widget front-end and back-end",
   "engines": {
-    "node": "5.x.x"
+    "node": "5.6.0"
```
Comment (JakeChampion) [inline package.json:13]: I think this may be better to leave with `5.x.x` as we can get the upgrades for free by redeploying the service on Heroku.

Comment (seanmtracey) [inline package.json:13]: I'd agree, I don't think there's ever a really good reason for locking down to a specific version number, unless there are known incompatibilities.

Comment (AdaRoseCannon) [inline package.json:13]: Heroku doesn't recommend having a floating version number on production sites. (Was reading the docs)

Comment (JakeChampion) [inline package.json:13]: Okay, then this looks good to me

**Pull request merged**